vars/tmsUpload.groovy

```diff
@@ -49,7 +50,8 @@ import static com.sap.piper.Prerequisites.checkScript
     /**
      * Proxy which should be used for the communication with the Transport Management Service Backend.
      */
-    'proxy'
+    'proxy',
+    'useGoStep'
```

Comment:
I think, it makes sense to provide a short description for this parameter, e.g. 
/**
     * Toggle to activate a new Golang implementation of the step. Off by default.
     * @possibleValues true, false
     */